src/local-server.js

```diff
@@ -37,34 +37,35 @@ fastify.get("/colleagues/:id", (req, res) => {
 });
 
 fastify.post("/colleagues", (req, res) => {
-  const colleague = JSON.parse(req.body);
+  const colleague = req.body;
```

Comment:
Note, `req.body` defaults to a string if the request does not specify `Content-Type: application/json`. Might want to add if-statements checking the header and manually deserialize the body or return `415 Unsupported Media Type` when content type is something else. Some of the previous participants were confused when seemingly correct requests didn't go through as they weren't aware that the header is neccessary.